src/server/tran_server.cpp

```diff
@@ -454,6 +454,21 @@ tran_server::uses_remote_storage () const
   return false;
 }
 
+bool tran_server::get_main_connection_info (std::string &host_out, int32_t &port_out)
+{
+  auto slock = std::shared_lock<std::shared_mutex> { m_main_conn_mtx };
+  if (m_main_conn == nullptr)
+    {
+      return false;
```

Comment:
Actually, more consistent to set the invalid host/port values here - in the same place where the valid values are set.
```suggestion
      host_out = "N/A";
      port_out = -1;
      return false;
```

Reply:
Thank you for the pointer. I'll keep it in mind.